core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -82,6 +83,16 @@ public LoggingDecoratorBuilder logger(Logger logger) {
         return this;
     }
 
+    /**
+     * Sets the specified loggerName which is used to create a new {@link Logger}.
```

Comment:
`Sets the name of the {@link Logger} to use when logging.` ?

Reply:
Could also add: `This method is a shortcut for `logger(LoggerFactory.getLogger(loggerName))`.

Reply:
Update done 😄